examples/specs/concat_bar_scales_discretize.vl.json

```diff
@@ -0,0 +1,123 @@
+{
+  "$schema": "https://vega.github.io/schema/vega-lite/v2.json",
+  "description": "A simple bar chart with embedded data.",
```

Comment:
Probably meant to have the description as "Horizontally concatenated charts that show different types of discretizing scales" instead?

Reply:
Also, noticed other examples description too, seem to be not updated.

Reply:
Good catch. Can you send a PR to fix these? Don't worry about updating the Vega and the images; they will update automatically.